app/lib/postcodeLinks.js

```diff
@@ -13,7 +13,7 @@ function filterNotFoundNames(name, address) {
   return `${name},${address}`;
 }
 
-function addUrl(postcodeLocationDetails, inputList) {
+function addMapUrl(postcodeLocationDetails, inputList) {
   return inputList.map((item) => {
     const address = joinAllTruthyValues(item.address);
     const saddr = `${postcodeLocationDetails.location.lat},${postcodeLocationDetails.location.lon}`;
```

Comment:
I know this hasn't changed but why is the lat/lon of the postcode being used for the start address rather than the postcode? Pharmacy finder uses the postcode as it is more obvious when clicking through to Google maps where the start address has come from. By using  the lat/lon the address is mapped to something else which might not be obvious as to how it was calculated.

Reply:
You can search both by outcode or by postcode - the lat long would cover both. I can raise an issue and either decompose the postcodeLocationDetails (different keys based on those) or use the users input. Which one do you suggest?

Reply:
Either outcode or postcode can be used for `saddr`. I'd go with setting it based on the input. Good idea to get an issue raised to cover this.

Reply:
Raised https://github.com/nhsuk/sexual-health-service-finder/issues/29 to cover this